lib/components/Body.js

```diff
@@ -50,6 +56,15 @@ var Body = React.createClass({
     if (isDate(body)) {
       return $(Data, null, body.toString());
     }
+    if (isSet(body)) {
+      var contents = Array.from(body)
```

Comment:
What I meant with `Don't need to use a package for that.` is that we can iterate on the Set very simply to create an Array from it.